medmodels/medrecord/datatype.py

```diff
@@ -212,8 +211,9 @@ class Union(DataType):
 
     def __init__(self, *dtypes: DataType) -> None:
         if len(dtypes) < 2:
-            raise ValueError("Union must have at least two arguments")
-        elif len(dtypes) == 2:
+            msg = "Union must have at least two arguments"
+            raise ValueError(msg)
```

Comment:
do you know which linting rule this is? Not sure if I like this

Reply:
When adding the string into the raise expression directly, it will be shown twice on error

https://docs.astral.sh/ruff/rules/raw-string-in-exception/

Reply:
But let's collect and discuss linting rules we might not wanna subscribe to

Reply:
with the explanation this rule makes sense